### PR TITLE
[docs] The example in tutorials doc missed checkpoint and primary key setting

### PR DESCRIPTION
**Pull request by frey66**

### CODE REVIEW

File: docs/content/tutorials/tutorial-zh.md [hunk 1]
```diff
@@ -136,18 +136,22 @@ VALUES (default,10001,'Beijing','Shanghai',false),
 
 
 4. 下载以下 jar 包到 `<FLINK_HOME>/lib/`:
-- [flink-sql-connector-elasticsearch7_2.11-1.11.1.jar](https://repo.maven.apache.org/maven2/org/apache/flink/flink-sql-connector-elasticsearch7_2.11/1.11.1/flink-sql-connector-elasticsearch7_2.11-1.11.1.jar)
-- [flink-sql-connector-mysql-cdc-1.0.0.jar](https://repo1.maven.org/maven2/com/alibaba/ververica/flink-sql-connector-mysql-cdc/1.0.0/flink-sql-connector-mysql-cdc-1.0.0.jar)
-- [flink-sql-connector-postgres-cdc-1.0.0.jar](https://repo1.maven.org/maven2/com/alibaba/ververica/flink-sql-connector-postgres-cdc/1.0.0/flink-sql-connector-postgres-cdc-1.0.0.jar)
+- [flink-sql-connector-elasticsearch7_2.12-1.13.2.jar](https://repo.maven.apache.org/maven2/org/apache/flink/flink-sql-connector-elasticsearch7_2.12/1.13.2/flink-sql-connector-elasticsearch7_2.12-1.13.2.jar)
```
Comment (leonardBang) [inline docs/content/tutorials/tutorial-zh.md:139]: nit: the `2.11` and `2.12` is the scala version, thus we do not need to change it

Comment (frey66) [reply docs/content/tutorials/tutorial-zh.md:139]: get it

File: docs/content/tutorials/tutorial-zh.md [hunk 2]
```diff
@@ -136,18 +136,22 @@ VALUES (default,10001,'Beijing','Shanghai',false),
...
-- [flink-sql-connector-postgres-cdc-1.0.0.jar](https://repo1.maven.org/maven2/com/alibaba/ververica/flink-sql-connector-postgres-cdc/1.0.0/flink-sql-connector-postgres-cdc-1.0.0.jar)
+- [flink-sql-connector-elasticsearch7_2.11-1.13.2.jar](https://repo.maven.apache.org/maven2/org/apache/flink/flink-sql-connector-elasticsearch7_2.11/1.13.2/flink-sql-connector-elasticsearch7_2.11-1.13.2.jar)
+- [flink-sql-connector-mysql-cdc-2.0.0.jar](https://repo1.maven.org/maven2/com/ververica/flink-sql-connector-mysql-cdc/2.0.0/flink-sql-connector-mysql-cdc-2.0.0.jar)
+- [flink-sql-connector-postgres-cdc-2.0.0.jar](https://repo1.maven.org/maven2/com/ververica/flink-sql-connector-postgres-cdc/2.0.0/flink-sql-connector-postgres-cdc-2.0.0.jar)
 
 5. 然后启动 Flink 集群，再启动 SQL CLI.
 
 ```sql
+-- 设置checkpoint间隔3秒                     
+Flink SQL> SET 'execution.checkpointing.interval' = '3s';
```
Comment (leonardBang) [inline docs/content/tutorials/tutorial-zh.md:147]: Could you add `Flink SQL> ` for all SQL statement ?

**Review by leonardBang — APPROVED** [review]
@frey66 Thanks for the nice work, I append a commit for minor improvement, LGTM

**Pull request merged**